system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -1371,6 +1371,27 @@ impl pallet_parameters::Config for Runtime {
 pub mod dynamic_params {
 	use super::*;
 
+	/// Parameters used to calculate staking era payouts.
+	#[dynamic_pallet_params]
+	#[codec(index = 0)]
+	pub mod inflation {
```

Comment:
Not sure if you saw Gavs recent interview, but he basically argued that *Inflation* is not the right terminology, but rather *Interest* or *Issuance*.

Reply:
noted and fixed.